src/index.js

```diff
@@ -0,0 +1,62 @@
+import './style.scss';
+import Arrow from './img/Arrows-Down-Left-icon.png';
+import Todo from './modules/Todo.js';
+
+function component() {
+  return `<div class='to-do'>
+    <div class='header border'>
+      <h1>Today's To Do</h1>
+      <span>
+        <i class='fa-solid fa-rotate'></i>
+      </span>
+    </div>
+    <div class="text-area border">
+    <div class='input-field'>
+      <input class="field description" type='text' placeholder='Add to your list' />
+      <img class="add-btn arrow-left" src=${Arrow} alt='' />
+    </div>
+    </div>
+    <div class='todo-list border'>
+     
+  </div>
+      <div class='clear-cmpl border'>
+        <button type="button">Clear all completed</button>
+      </div>`;
+}
+document.querySelector('.container').innerHTML = component();
+
+// Global variables
+const items = [];
+const btn = document.querySelector('.add-btn');
+const container = document.querySelector('.todo-list');
+
+// Event handler
+
+btn.addEventListener('click', () => {
+  const id = Date.now();
+  const desc = document.querySelector('.description');
+  const description = desc.value;
+  const completed = false;
+  const newTodo = new Todo(id, description, completed);
+  items.push(newTodo);
+  addtoDom();
+});
+
```

Comment:
- [ ] I think here the requirements is to make a static structure  for the todolist so please check the requirements and implement it as required, I know that you made a huge efforts with adding dynamically but the required is just to display some tasks and what you did now this is for the next milestone 💪🏻💯💥